langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -116,6 +158,10 @@ public isolated function indexOf(string str, string substr, int startIndex = 0)
 
 # Finds the last occurrence of one string in another string.
 #
+# ```ballerina
+# int? lastIndex = "There are multiple programming languages. Ballerinalang is unique among them".lastIndexOf("lang");
```

Comment:
Shall we wrap up this string to a variable. wdyt?

Reply:
shall we use a shorter sentence for this? so then the user may not need to scroll to the right to see the whole content
e.x: 

```ballerina
string description = "Ballerinalang is a unique programming language";
int? lastIndex = lastIndexOf("lang");
```